docs/lib/geo/fragments/js/existing-resources.md

```diff
@@ -0,0 +1,33 @@
+## In your application
+
+To configure Geo manually, you will have to configure Amplify Auth category as well. In other words, you will not be importing the autogenerated `aws-exports.js` - instead, you will be adding your own existing Amazon Cognito and Amazon Location Service credentials in your app like this:
+
+```js
+import Amplify, { Auth, Geo } from 'aws-amplify';
+
+Amplify.configure({
+  Auth: {
+    identityPoolId: 'XX-XXXX-X:XXXXXXXX-XXXX-1234-abcd-1234567890ab', // REQUIRED - Amazon Cognito Identity Pool ID
+    region: 'XX-XXXX-X', // REQUIRED - Amazon Cognito Region
+    userPoolId: 'XX-XXXX-X_abcd1234', // OPTIONAL - Amazon Cognito User Pool ID for authenticated user access
+    userPoolWebClientId: 'XX-XXXX-X_abcd1234', // OPTIONAL - Amazon Cognito Web Client ID for authenticated user access
+  },
+  Geo: {
+    AmazonLocationService: {
```

Comment:
Does this work? I thought we needed `geo` and `amazon_location_services` as the keys

Reply:
Spoke on this offline. It's primarily the `geo/Geo` part right now that effects this. Will update to `geo`, but leave `AmazonLocationServices` for now. Will be having a conversation with other JS devs on if this needs to stay consistent with other categories as far as the casing goes and we will update as/if needed.